src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -2837,22 +2837,27 @@ subroutine get_GEOSs2s(date_time, met_path, met_tag, N_catd, tile_coord, &
     
     do GEOSgcm_var = 1,N_GEOSgcm_vars
        
-       if (GEOSgcm_var==1) then
-          ! init share memory
-          if(  (size(ptrShForce,1) /= GEOSgcm_grid_N_lon) .or.          &
-               (size(ptrShForce,2) /= GEOSgcm_grid_N_lat)       ) then
-             call MAPL_SyncSharedMemory(rc=status)
-             VERIFY_(status)
-             if (associated(ptrShForce)) then
-                call MAPL_DeallocNodeArray(ptrShForce,rc=status)
-                VERIFY_(status)
-             endif
-             call MAPL_AllocateShared(ptrShForce,(/GEOSgcm_grid_N_lon,GEOSgcm_grid_N_lat/),TransRoot= .true.,rc=status)
-             VERIFY_(status)
-             call MAPL_SyncSharedMemory(rc=status)
+       ! init shared memory
+       N_lon_tmp = -1
+       N_lat_tmp = -1
+       if (associated(ptrShForce)) then
+          N_lon_tmp = size(ptrShForce,1)
+          N_lat_tmp = size(ptrShForce,2)
+       endif
+       if(  (size(ptrShForce,1) /= GEOSgcm_grid_N_lon) .or.          &
+            (size(ptrShForce,2) /= GEOSgcm_grid_N_lat)       ) then
+          call MAPL_SyncSharedMemory(rc=status)
+          VERIFY_(status)
+          if (associated(ptrShForce)) then
+             call MAPL_DeallocNodeArray(ptrShForce,rc=status)
              VERIFY_(status)
-          end if
-       endif ! (GEOSgcm_var==1)
+          endif
+          call MAPL_AllocateShared(ptrShForce,(/GEOSgcm_grid_N_lon,GEOSgcm_grid_N_lat/),TransRoot= .true.,rc=status)
+          VERIFY_(status)
+          call MAPL_SyncSharedMemory(rc=status)
+          VERIFY_(status)
+       end if
+
        rc = 0
        call MAPL_SyncSharedMemory(rc=status)
```

Comment:
Do we need these lines?  They seem redundant with the final two lines in the preceding if block.

Reply:
Cleaned up with [`15c2763`](https://github.com/GEOS-ESM/GEOSldas/pull/442/commits/15c276386493b4304696f8af723d08467120e6a4)